homeassistant/components/http/view.py

```diff
@@ -102,8 +102,8 @@ def request_handler_factory(view, handler):
             else:
                 raise HTTPUnauthorized()
 
-        _LOGGER.info('Serving %s to %s (auth: %s)',
```

Comment:
You can't justify changes to a core component to fix noisy logs.

Reply:
Fair enough.

Still; even though my initial reason to look into this was noisy logs, I believe `info` is a too high level for this kind of message.
While the frontend is not of *primary* importance for Home Assistant, it is never the less one of the most widely used components.
Serving a http request is, as I see it, something that should be handled and thought of at the same level as, say, an event or a state change.

It's simply not interesting enough to the average user as an event to warrant the `info` level.
And having to sift through endless streams of `2019-02-16 00:25:19 INFO (MainThread) [homeassistant.components.http.view] Serving /api/history/period/2019-02-15T11:01:12.133Z to 172.22.0.1 (auth: True)` or adding manual overrides (something that's not quite obvious how to do for a beginner) of the log level for the component in order to find what you actually want from the log is not very user friendly.

But perhaps you are saying this should have been discussed first in some other repo which I can't find at the moment?

Reply:
Default logger level is Warning. normal user won't be bothered by this log.

Reply:
![skarmavbild 2019-02-16 kl 07 38 09](https://user-images.githubusercontent.com/1299821/52895670-ead63e80-31bd-11e9-93e5-d3ca280bf56a.png)

Reply:
The docs says `debug` is default, btw. https://www.home-assistant.io/components/logger/#default